README.md

```diff
@@ -588,18 +655,20 @@ The parameters passed to the function are:
 
 
 
-##### `set`
+#### `set`
 
 A function that is called whenever the property is set. This is also called when the property is first initialised.
 
 ```js
-skate.define('my-component', {
-  props: {
-    myProp: {
-      set (elem, data) {
-        // do something
+customElements.define('my-component', class extends skate.Component {
+  static get props () {
+    return {
+      myProp: {
+        set (elem, data) {
+          // do something
```

Comment:
Could you please mention this issue (https://github.com/skatejs/skatejs/pull/866#pullrequestreview-4409414) here, if this is still the case?